db/schema.cds

```diff
@@ -71,6 +71,7 @@ entity BookingSupplement : managed {
 //
 
 entity BookingStatus : CodeList {
+  @cds.collate: false
```

Comment:
Maybe we should have some heuristic to default to collate=false for enums.

Reply:
And for keys maybe?